test/e2e/performanceprofile/functests/8_performance_workloadhints/test_suite_performance_workloadhints_test.go

```diff
@@ -0,0 +1,51 @@
+//go:build !unittests
+// +build !unittests
+
+package __performance_workloadhints_test
```

Comment:
if memory serves this is the only functional test file with the `_test` suffix, why so?

Reply:
right, I was remembering wrongly.